workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileTypeCollection.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    [CreateAssetMenu(fileName = "New Prefab Collection", menuName = "Improbable/Create Prefab Collection")]
```

Comment:
Asset and menu name are less specific than what this is.

`Prefab Collection` vs `Tile Collection`

Reply:
Good catch, some legacy stuff from the script's old lifestyle as a generic prefab collection there.